tests/integration/conftest.py

```diff
@@ -11,12 +11,12 @@ def client(ert_storage_client):
 
 @pytest.fixture
 def create_ensemble(client):
-    def func(experiment_id, parameters=None, update_id=None):
+    def func(experiment_id, parameters=None, update_id=None, size=-1):
```

Comment:
Just want to know the reasoning: Why `-1` and not set is as `nullable` and have it `None` when `size` was not provided?

Reply:
Since we don't want this to be valid behaviour in the future. Once ERT supports POSTing the size, we'll add checks that verify that the size is correct. Right now we don't, but why make it nullable now when we'll make it non-nullable in the short term future?

Reply:
Ok, what I meant is that the endpoints which (will) make use of `ensemble size` will just test `if ensemble_size is not None` while now we would do `if ensemble_size!=-1`. I thought that `is not None` is just more pythonic :) That's it

Reply:
I think - if i'm not mistaken this is a built-in and the request will fail if `None` is provided. At least that was what i was thinking and thus just sent `-1` in the tests as a standard which applies for me at least to be a not-valid size.